contracts/payloads/IGP85/PayloadIGP85.sol

```diff
@@ -0,0 +1,548 @@
+                baseBorrowLimitInUSD: 10_000_000, // $10M
+                maxBorrowLimitInUSD: 20_000_000 // $20M
+            });
+
+            setVaultLimits(VAULT_ezETH_wstETH); // TYPE_1 => 103
+
+            VAULT_FACTORY.setVaultAuth(
+                ezETH__wstETH_VAULT,
+                TEAM_MULTISIG,
+                false
+            );
+        }
+
+        {
+            address ezETH_ETH__wstETH_VAULT = getVaultAddress(104);
+
+            // [TYPE 2] ezETH-ETH<>wstETH | smart collateral & normal debt
+            Vault memory VAULT_ezETH_ETH_wstETH = Vault({
+                vault: ezETH_ETH__wstETH_VAULT,
+                vaultType: TYPE.TYPE_2,
+                supplyToken: address(0),
+                borrowToken: wstETH_ADDRESS,
+                baseWithdrawalLimitInUSD: 0,
+                baseBorrowLimitInUSD: 7_500_000, // $7.5M
+                maxBorrowLimitInUSD: 15_000_000 // $15M
+            });
+
+            setVaultLimits(VAULT_ezETH_ETH_wstETH); // TYPE_2 => 104
+
+            VAULT_FACTORY.setVaultAuth(
+                ezETH_ETH__wstETH_VAULT,
+                TEAM_MULTISIG,
+                false
+            );
+        }
+    }
+
+    // @notice Action 4: Update rewards for fUSDC, fUSDT
+    function action4() internal isActionSkippable(4) {
+        address[] memory protocols = new address[](2);
+        address[] memory tokens = new address[](2);
+        uint256[] memory amounts = new uint256[](2);
+
+        address REWARDS_ADDRESS = address(
+            //0x
+        );
+
+        {
+            /// fUSDC
+            IFTokenAdmin(F_USDC).updateRewards(REWARDS_ADDRESS);
+
+            uint256 allowance = IERC20(USDC_ADDRESS).allowance(
+                address(FLUID_RESERVE),
+                F_USDC
+            );
+
+            protocols[0] = F_USDC;
+            tokens[0] = USDC_ADDRESS;
+            amounts[0] = allowance ;
+        }
+
+        {
+            /// fUSDT
+            IFTokenAdmin(F_USDT).updateRewards(REWARDS_ADDRESS);
+
+            uint256 allowance = IERC20(USDT_ADDRESS).allowance(
+                address(FLUID_RESERVE),
+                F_USDT
+            );
+
+            protocols[1] = F_USDT;
+            tokens[1] = USDT_ADDRESS;
+            amounts[1] = allowance ;
+        }
+
+        FLUID_RESERVE.approve(protocols, tokens, amounts);
+
+        {
+            ILendingRewards(REWARDS_ADDRESS).start();
+        }
+    }
+
+    // @notice Action 5: Constrict BOLD DEX
+    function action5(){
+
+        {
+            address USDC_BOLD_DEX = getDexAddress(25);
+            // USDC-BOLD DEX
+            {
+                // USDC-BOLD Dex
+                Dex memory DEX_USDC_BOLD = Dex({
+                    dex: USDC_BOLD_DEX,
+                    tokenA: USDC_ADDRESS,
+                    tokenB: BOLD_ADDRESS,
+                    smartCollateral: true,
+                    smartDebt: false,
+                    baseWithdrawalLimitInUSD: 10, // $10
```

Comment:
in USD amounts we can even make it `1`

Reply:
can't use standard expansion here when setting dex limits, should be 
expandPercent: `1`
expandDuration: `16777215`
baseWithdrawalLimit: `10`